assets/src/scripts/components/map/index.js

```diff
@@ -20,11 +20,29 @@ const getLayerDefs = function(layerNo, siteno, stage) {
  * Creates a site map
  */
 const siteMap = function(node, {siteno, latitude, longitude, zoom}) {
+
+    var gray = layerGroup();
```

Comment:
Use 'let' rather than 'var'. We are using ES6 in waterdataui

Reply:
fixed